packages/ui-alerts/src/Alert/index.js

```diff
@@ -228,13 +228,20 @@ class Alert extends Component {
       const div = document.getElementById(this.srid)
       if (div) {
         ReactDOM.render(null, div, () => {
-          const content = this.createScreenreaderContentNode()
-          ReactDOM.render(content, div)
+          this.renderScreenreeaderAlert(div)
         })
       }
     }
   }
 
+  renderScreenreeaderAlert(div) {
+    const content = this.createScreenreaderContentNode()
+    ReactDOM.render(
+      <EmotionThemeProvider theme={canvas}>{content}</EmotionThemeProvider>,
+      div
+    )
```

Comment:
why is the `canvas` theme hardcoded here?

Reply:
Well, the thing is, the `ScreenReaderContent` that gets generated inside doesn't really need a theme, but since this block gets rendered outside of our app (it has its own `ReactDOM.render()`), it puts a warning on the console that "no themes found, applying default canvas theme". Other option would be to pass it an "empty" theme, or trying to refactor it completely. Would you prefer the empty theme?

Reply:
no, I think this is better, I guess an empty theme might cause issues (e.g. missing properties). Maybe add a comment here explaining this edge case.

Reply:
added the comment, thanks :)